src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/SyncRequest.kt

```diff
@@ -49,6 +49,10 @@ data class UpsertSyncRequest(
   @field:Size(min = 1, max = 60, message = "Authorised by must be a maximum of 60 characters")
   val authorisedBy: String? = null,
 
+  @Schema(description = "The last staff member who changed this record, username of 30 characters, if not provided the API credentials will be used", required = false, example = "JSMITH", maxLength = 30)
+  @field:Size(min = 1, max = 30, message = "Authorised by must be a maximum of 60 characters")
+  val lastModifiedByUsername: String? = null,
```

Comment:
New field (optional - this could be changed to mandatory once syscon is already sending it)